package.json

```diff
@@ -4,6 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "build": "tsc",
+    "prod": "knex migrate:latest && node dist/src/deploy-commands.js && node dist/src/index.js",
```

Comment:
I have no idea why I had to do `dist/src` here instead of just `dist` like below. The Dockerfile is running `tsc` in the repo root (the parent directory of `src` to be exact).